go.sum

```diff
@@ -278,6 +278,8 @@ github.com/vmihailenco/msgpack v4.0.1+incompatible h1:RMF1enSPeKTlXrXdOcqjFUElyw
 github.com/vmihailenco/msgpack v4.0.1+incompatible/go.mod h1:fy3FlTQTDXWkZ7Bh6AcGMlsjHatGryHQYUTf1ShIgkk=
 github.com/vmware/vra-sdk-go v0.2.11 h1:QSnlKItvEmnhd9yfaCLesTlb/I928F/OhQ7LpaGljyI=
 github.com/vmware/vra-sdk-go v0.2.11/go.mod h1:W6OERth9ssa4FgvS98SOVusGgBNWJs+D4Hq/5ueTgIQ=
+github.com/vmware/vra-sdk-go v0.2.12 h1:kn6EMcIsOxWG0s7JjWWBGihpzP+wGLqvXSSk83wH+TU=
+github.com/vmware/vra-sdk-go v0.2.12/go.mod h1:W6OERth9ssa4FgvS98SOVusGgBNWJs+D4Hq/5ueTgIQ=
```

Comment:
Run `go mod tidy` to clean up the older versions.

Reply:
Thanks, cleaned it.